src/components/profile/DataClassification.vue

```diff
@@ -95,7 +131,13 @@ export default {
   color: #594300;
 }
 
-.data-classification__label {
+.data-slack__access {
+  background-color: #611f69;
+  color: #fff;
+}
+
+.data-classification__label,
+.data-slack__access__label {
```

Comment:
In terms of CSS naming we try to follow http://getbem.com/naming/ but these should change if we refactor that  component anyhow.